themes/default/content/what-is/what-are-circleci-secrets.md

```diff
@@ -0,0 +1,154 @@
+---
+title: What are CircleCI secrets?
+meta_desc: |
+     Learn more about what CircleCI Secrets are and how to use them.
+
+type: what-is
+page_title: "What are CircleCI secrets?"
+---
+
+[CircleCI](https://circleci.com/) is an agile Continuous Integration/Continuous Deployment (CI/CD) platform. It aims to automate software development processes for faster, more reliable releases. CircleCI secrets empower developers to safeguard critical data while streamlining workflows.
+
+## What are CircleCI secrets?
+
+CircleCI secrets are a secure and efficient means of managing sensitive data within your CI/CD pipelines. These secrets encompass API keys, passwords, and other confidential data crucial for the successful execution of your build and deployment processes.
```

Comment:
in this case CircleCI secrets isn't a Product per say, but rather a means. What about something like: CircleCI secrets are sensitive pieces of information that are necessary for building and deploying applications securely within your CI/CD pipeline. They can include things like API tokens, SSH keys, and environment variables that contain credentials. These are meant to be hidden from the public and even from certain team members to maintain security and integrity of the access they provide. In CircleCI, secrets are used within the configuration of continuous integration and delivery pipelines to allow automated processes to interact with other services, repositories, and infrastructure securely. Managing these secrets securely is crucial to prevent unauthorized access and potential security breaches.

Reply:
Thank you, this makes it much clearer :)